updater.js

```diff
@@ -0,0 +1,252 @@
+var path = require('path')
+var fs = require('fs')
+var childProcess = require('child_process')
+var pkgUp = require('pkg-up')
+
+var BrowserslistError = require('./error')
+
+var UNKNOWN_VERSION = 'unknown version'
+var PACKAGE_CANIUSE = 'caniuse-lite'
+
+var packageManager = ''
+
+function updateDB () {
+  var lockfile = getLockfile()
+
+  fs.readFile(lockfile, 'utf8', function (readError, lockfileRaw) {
```

Comment:
You use sync methods in one case and async methods in another. Let’s be consistent here.

I think sync methods are OK here since there will be no other task run in the same Node.js instance (async tasks are good when you have many different tasks, like in Gulpfile or web server)